src/plugins/owlexporter/CExporterOwl.cpp

```diff
@@ -152,7 +152,63 @@ namespace semrec {
 	    }
 	  }
 	}
+
+	//m_lstAnnotationPurposeMapping.clear();
 	
```

Comment:
Did you copy the contents of the `cfgConfig.exists(...)` block? I don't quite see why at the moment. If you need to add another semantic annotation to the `semantics_descriptor_files` file, we should find a more general way.